integration/integration_test.go

```diff
@@ -206,18 +208,21 @@ func TestServerStarts(t *testing.T) {
 }
 
 func TestServerFailsToStart(t *testing.T) {
-	var config strings.Builder
-
-	_, _ = config.WriteString("server:\n")
-	_, _ = config.WriteString("  strict_mode: false\n")
-	_, _ = config.WriteString("  port: 50052\n")
-	_, _ = config.WriteString("  tls: not_boolean\n")
-	_, _ = config.WriteString("logging:\n")
-	_, _ = config.WriteString("  level: debug\n")
-	_, _ = config.WriteString("  selectors: [\"*\"]\n")
-	_, _ = config.WriteString("  to_stderr: true\n")
-
-	env := NewTestingEnvironment(t, config.String())
+	config := `
+server:
+  strict_mode: false
+  port: 50052
+  tls: not_boolean
+logging:
+  level: debug
+  selectors: ["*"]
+  to_stderr: true
+output:
+  console:
+    enabled: true
```

Comment:
Looks like this is the only change in all test configs, right?

Reply:
Yes, a while back the shipper started quitting with an error if no outputs are active but the integration test wasn't updated to include it